string start_floor
string destination_floor
string start_area
string destination_area
string start_local_area
string destination_local_area
Point start_position
string destination_task
BlockedConnection[] blocked_connections
bool relax_traffic_rules
---
PlannerArea[] planner_areas
---